contracts/governance.clar

```diff
@@ -49,19 +49,6 @@
     )
 )
 
-;; Cancels an existing timelock by setting its eta to zero.
-;; @params hash; The hash of the timelock to cancel
-;; @returns (response true) or reverts
-(define-private (cancel-timelock (hash (buff 32)))
```

Comment:
Like I said in the issue, keep this private function and call it from `execute` if the payload type is `ACTION-CANCEL-TASK`.

From what I see, this would require refactoring `execute` a bit to first get only the `type` from the payload. If it is `ACTION-CANCEL-TASK` then you can get the `hash` from the payload, if not then the current logic will be invoked.